services/QuillLMS/client/app/bundles/Staff/components/newAdminOrDistrictUser.tsx

```diff
@@ -53,7 +68,8 @@ const NewAdminOrDistrictUser = ({ type, returnUrl, schoolId, districtId }: NewAd
       const params = {
         first_name: firstName.trim(),
         last_name: lastName.trim(),
-        email: email.trim()
+        email: email.trim(),
+        school_ids: checklistSchools.filter((s) => s.checked).map((s) => s.id),
```

Comment:
NBD but for styling consistency, we usually don't wrap single parameters in parentheses (as you don't elsewhere on this page), so I would write these as `(s => s.checked)` and `(s => s.id)`.

Reply:
It looks like we could use an eslint [rule](https://eslint.org/docs/latest/rules/arrow-parens) for that, specifically the `as-needed` config